src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -37,6 +37,11 @@ the MIT License. See LICENSE in the project root for license information. -->
                 <Setter Property="Margin" Value="0,10,0,0"/>
             </Style>
 
+            <Style TargetType="TextBlock">

```

Comment:
Add an `x:Key` and reference it whenever you want to use it.

Reply:
Wait, isn't there a style that already exists? Some kind of header or subheader style?

Reply:
I'd also move this style to CommonResources.xaml. That way we keep all of them together in one place. (Assuming there isn't a style that already exists)